settings.py

```diff
@@ -553,6 +553,9 @@ def JINJA_CONFIG():
 STANDALONE_DOMAINS = ['django', 'javascript']
 STATICI18N_DOMAIN = 'javascript'
 
+# Cache non-versioned static files for one week
+WHITENOISE_MAX_AGE = 604800
```

Comment:
Would you mind writing that in the form of `60 * 60 * 24 * 7`? That has in my experience removed the magic behind such numbers and prevented confusion when others start working on the project.